common/hash.go

```diff
@@ -36,4 +36,4 @@ func (a *Hash) Bytes() []byte {
 
 func (a *Hash) String() string {
 	return string(a[:])
-}
+}
```

Comment:
take care this warn, nice to add new line.

Reply:
I use goland, it doesn't show warning. anyway, I will add new line